lib/puppet/provider/package/yum.rb

```diff
@@ -9,7 +9,7 @@
   These options should be specified as a string (e.g. '--flag'), a hash (e.g. {'--flag' => 'value'}),
   or an array where each element is either a string or a hash."
 
-  has_feature :install_options, :versionable, :virtual_packages
+  has_feature :install_options, :versionable, :virtual_packages, :package_arch
```

Comment:
None of the package ones currently follow that standard — see http://docs.puppetlabs.com/references/latest/type.html#package-provider-features for the list. Since the standard doesn't seem to be universal, I don't particularly care, and would default to using the local style for that resource type. (In this case, using the name of the parameter as the name of the feature.)